{
  try {
    return method.invoke(mbs,args);
  }
 catch (  InvocationTargetException e) {
    Throwable t=e.getCause();
    throw t;
  }
}
